uobtheatre/bookings/test/test_mutations.py

```diff
@@ -2463,7 +2463,7 @@ def test_check_in_booking_fails_if_not_paid(gql_client, status):
         performance=performance, user=gql_client.user, status=status
     )
 
-    checked_in_ticket = TicketFactory(booking=booking, checked_in=True)
+    checked_in_ticket = TicketFactory(booking=booking, checked_in_at=timezone.now())
```

Comment:
Part of the mutation logic that you introduce with this PR is that you pass the current logged in user as the user who checked in the ticket (https://github.com/BristolSTA/uobtheatre-api/pull/607/files#diff-6d10df12e83f25dd8b0a9a9df90593da84dd0a31699625bb5aec191948fb6bdcR350). This isn't tested anywhere in this file. You should add an assertion in the `test_check_in_booking` (https://github.com/BristolSTA/uobtheatre-api/blob/263b195337eea082f872ce158180a7d206d7ee57/uobtheatre/bookings/test/test_mutations.py#L2284) method that the current GQL user is set as the checked_in_by user on the ticket